installers-conda/resources/post-install.sh

```diff
@@ -106,37 +118,54 @@ if [[ -n "$shell_init" ]]; then
 fi
 
 # ----
-if [[ $OSTYPE = "linux"* ]]; then
+if [[ "$OSTYPE" = "linux"* ]]; then
     cat <<EOF
 
+###############################################################################
+#                             !!! IMPORTANT !!!
 ###############################################################################
 Spyder can be launched by standard methods in Gnome and KDE desktop
-environments. Additionally, Spyder can be launched in Gtk-based desktop
-environments (e.g. Xfce) from the command line:
-
-$ gtk-launch spyder
-
-Spyder can also be launched from the command line for all Linux variants by:
+environments. It can also be launched from the command line on all Linux
+distros with the command:
 
 $ spyder
 
 To uninstall Spyder, run the following from the command line:
 
 $ uninstall-spyder
 
-#####################
-# !!! IMPORTANT !!! #
-#####################
-
-The spyder and uninstall-spyder commands will only be available in new shell
-sessions. To make them available in this session you must source your .bashrc
-file with:
+These commands will only be available in new shell sessions. To make them
+available in this session, you must source your $shell_init file with:
 
-$ source ~/.bashrc
+$ source $shell_init
 
 ###############################################################################
 
 EOF
 fi
 
 echo "*** Post install script for ${INSTALLER_NAME} complete"
+
+# ----
+[[ -n "$CI" ]] && exit 0  # Running in CI, don't launch Spyder
+
+if [[ "$OSTYPE" = "darwin"* ]]; then
```

Comment:
@ccordoba12, how about this instead? macOS install log can have the output too.
```suggestion
echo "Launching Spyder now..."
if [[ "$OSTYPE" = "darwin"* ]]; then
```

Reply:
Sure, that's fine for me.